CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,18 @@
 
 import Foundation
 
-print("Hello, World!")
+func printicecream(lengthoficecream: Int) {
+    for _ in 1 ... lengthoficecream {
+    print("***********")
+}
+}
+func printbar(lengthofbar: Int) {
+    for _ in 1...lengthofbar {
+    print("    | |    ")
+}
+}
+    printicecream(lengthoficecream: 8)
+    printbar(lengthofbar: 4)
```

Comment:
```suggestion
    printicecream(of: 8)
    printbar(of: 4)
    
    func printicecream(of length: Int) {
        for _ in 1 ... length {
        }
    }     
```

이미 메소드 이름에 `iceCream` 또는 `bar`가 들어가는데 매개변수에도 동일한 단어가 들어가서 이상하네요. 이렇게 바꿔보는건 어떨까요?

Reply:
오호 좀 더 효율적으로 코딩하네요 감사합니다.